caddyhttp/basicauth/setup_test.go

```diff
@@ -70,6 +70,30 @@ md5:$apr1$l42y8rex$pOA2VJ0x/0TwaFeAF9nX61`
 		}`, false, "pwd", []Rule{
 			{Username: "user", Resources: []string{"/resource1", "/resource2"}},
 		}},
+		{`basicauth user pwd {
+			path /resource1
+			path /resource2
+		}`, false, "pwd", []Rule{
+			{Username: "user", Resources: []string{"/resource1", "/resource2"}},
+		}},
+		{`basicauth user pwd {
+			/resource1
+			realm "Secure resources"
+			path /resource2
```

Comment:
One additional note, the example you highlight showing both the isolated resource and the one preceded by the `path` subdirective is not intended as an example, just as a test to show that both forms are supported. I would hope that going forward, people would use the `path` and `realm` subdirectives.